jenkins/release.jenkinsFile

```diff
@@ -14,4 +14,18 @@ standardReleasePipelineWithGenericTrigger(
             mavenArtifactsPath: "$WORKSPACE/repository/",
             autoPublish: true
         )
+        publishToArtifactsProdBucket(
+            assumedRoleName: 'sql-jdbc-upload-role',
+            source: "$WORKSPACE/artifacts.tar.gz",
+            destination: "https://artifacts.opensearch.org/opensearch-clients/jdbc/opensearch-sql-jdbc-${tag}.tar.gz",
+            signingPlatform: 'windows',
+            sigOverwrite: true
+        )
+        publishToArtifactsProdBucket(
+            assumedRoleName: 'sql-jdbc-shadowJar-upload-role',
```

Comment:
they have a separate role for shadowJars?

Reply:
I'm not actually too sure this works. I assumed that it was just a name for the jenkins job.
`@param Map[assumedRoleName] <required> - IAM role to be assumed for uploading artifacts`
Is from https://github.com/opensearch-project/opensearch-build-libraries/blob/de8ea9024a1a95a72c668562cbdd13065bf1ed46/vars/publishToArtifactsProdBucket.groovy#L12C1-L12C88 where that is being called.